Core/Executor/RepositoryExecutor.php

```diff
@@ -100,7 +100,7 @@ public function execute(MigrationStep $step)
 
             $previousUserId = $this->loginUser(self::ADMIN_USER_ID);
             try {
-                $output = $this->$action();
+                $output = $this->$action($step->dsl);
```

Comment:
It is available, but we can't use a stateful variable (explained that in the commit message). Since all executors have the DSL stored as a property, changes by one affect the DSL read by the other layers. And bam.

It is kind of a big change, but I don't see big counterparts.